htdocs/web_portal/controllers/service_type/view_service_type.php

```diff
@@ -20,12 +21,14 @@
  * limitations under the License.
  /*======================================================*/
 require_once __DIR__ . '/../utils.php';
-require_once __DIR__ . '/../../../web_portal/components/Get_User_Principle.php';
+require_once __DIR__
+    . '/../../../web_portal/components/Get_User_Principle.php';
```

Comment:
Is it possible to use parentheses around `__DIR__ . '/../../../web_portal/components/Get_User_Principle.php'` to make it clear this is the same argument to `require_once`.

(Note, I am not a PHP expert! I don't know if that is a good idea or not!)